ansible_builder/steps.py

```diff
@@ -58,8 +59,9 @@ def __init__(self):
         self.steps = []
         self.steps.extend([
             "",
-            "COPY --from=galaxy {0} {0}".format(constants.base_roles_path),
-            "COPY --from=galaxy {0} {0}".format(constants.base_collections_path),
+            "COPY --from=galaxy {0} {0}".format(
+                os.path.dirname(constants.base_collections_path.rstrip('/'))  # /usr/share/ansible
+            ),
```

Comment:
I actually think we can also remove line 51 above, there is no longer the need to create the collections / roles folders, as we can default to ansible-galaxy doing it.

Reply:
@AlanCoding making sure you seen this comment.

Reply:
Yeah, I was still testing rebuilding the awx-ee and the blank example. It looks like both are fine with this change, so I'll push a commit.